dev_tools/docs/nxdl.py

```diff
@@ -12,10 +12,15 @@
 from ..globals.errors import NXDLParseError
 from ..globals.nxdl import NXDL_NAMESPACE
 from ..globals.urls import REPO_URL
+from ..utils import nxdl_utils as pynxtools_nxlib
```

Comment:
Can you avoid the use of the alias? Just import and use `get_inherited_nodes`.

Reply:
DONE